package-lock.json

```diff
@@ -4975,9 +5070,9 @@
       }
     },
     "caniuse-lite": {
-      "version": "1.0.30001194",
-      "resolved": "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001194.tgz",
-      "integrity": "sha512-iDUOH+oFeBYk5XawYsPtsx/8fFpndAPUQJC7gBTfxHM8xw5nOZv7ceAD4frS1MKCLUac7QL5wdAJiFQlDRjXlA=="
+      "version": "1.0.30001251",
```

Comment:
Every once in a while, npm displays a message that "supported browsers list" needs to be updated (a npx command). So I do it, and it results in this change.